bridge/cypress/support/pageobjects/SequencesPage.ts

```diff
@@ -104,8 +104,13 @@ export class SequencesPage {
     return this;
   }
 
-  public selectSequence(keptnContext: string): this {
-    cy.byTestId(`keptn-root-events-list-${keptnContext}`).click();
+  public selectSequence(keptnContext: string, stage?: string): this {
+    const rootEvent = cy.byTestId(`keptn-root-events-list-${keptnContext}`);
+    if (stage) {
+      rootEvent.find('dt-tag').contains(stage).parentsUntil('ktb-stage-badge').first().click({ force: true });
```

Comment:
Is `force: true` needed here? For me this seems like a workaround because of the "detached from DOM" error

Reply:
Seems like I used `parentsUntil` instead of `parents`